swaybar/render.c

```diff
@@ -462,9 +510,11 @@ void render_frame(struct swaybar *bar, struct swaybar_output *output) {
 		cairo_set_source_surface(shm, recorder, 0.0, 0.0);
 		cairo_paint(shm);
 
+		wl_surface_set_buffer_scale(output->surface, output->scale);
 		wl_surface_attach(output->surface,
 				output->current_buffer->buffer, 0, 0);
-		wl_surface_damage(output->surface, 0, 0, output->width, output->height);
+		wl_surface_damage(output->surface, 0, 0,
+				output->width, output->height);
 		wl_surface_commit(output->surface);
 		wl_display_roundtrip(bar->display);
```

Comment:
I still don't get why this roundtrip is needed?

Reply:
We don't dispatch in a loop like some of our other clients do.

Reply:
Oh, I see, makes sense